### DISCUSSION TITLE
Can we target an action in pathless route?

### DISCUSSION THREAD

**Discussion by otatar** [remix-run/remix#8125] [Q&A]
He there, I have one question? Can we target an action in pathless route (filename prefix is _). For example, suppose we have routes like this:
```
_layout/
  route.tsx
  profile.tsx
```
route.tsx is a route, and profile.tsx is just a module. If profile.tsx has a form, is it possible to target an action  function in route.tsx?
```
<Form method="post" action="?"
```

**Comment by gustavopch** [comment]
I guess that route would be essentially at the root, i.e. `/`, but haven't tested.

**Reply by otatar** [reply]
If we put action="/" we then target root.tsx
